testgen/generators/datetime_gen.js

```diff
@@ -260,6 +285,19 @@ function generateAll(run_limit) {
           all_options['timeZone'] = timezone;
         }
         if (number_system) {
+          try {
+            const common_number_systems = intl_locale.numberingSystems;
+            const supported_calendars = intl_locale.calendars;
+            // TODO: Check if the number system is supported in this locale.
```

Comment:
Is it easy enough to implement the code for the TODO task re: the number system?

Reply:
Updated this. PTAL.